### ISSUE TITLE
Reader model training config for WQ and TREC

### ISSUE THREAD

**Issue by robinsongh381** [facebookresearch/DPR#145]
@vlad-karpukhin 
Hello !

When you train a reader model for WQ and TREC, what was yout training config ?  Especially how many epochs did you train with and learning rate?

**Comment by vlad-karpukhin** [comment]
Hi @robinsongh381,
It was Sewon (https://shmsw25.github.io/) who did WQ and TREC single reader training. I'd suggest to email her this question.
Also, FYI, our reader model is quite old, far from current reader SOTA models.
